database/migrations/default/1750435376415_ems_person_match_auto/up.sql

```diff
@@ -459,7 +487,36 @@ BEGIN
 END;
 $function$;
 
--- delete this trigger which is now redundant
+
+--
+-- This is a new function that triggers EMS re-matching when a person record is inserted or updated
+--
+CREATE
+OR REPLACE FUNCTION public.people_dispatch_ems_match() RETURNS trigger LANGUAGE plpgsql AS $function$
+DECLARE
+    person_crash_pk integer;
+BEGIN
+    raise debug '**function: people_dispatch_ems_match **';
+    
+    SELECT units.crash_pk INTO person_crash_pk FROM units where units.id = NEW.unit_id;
+    
+    raise debug 'Dispatching update event to EMS records matched to crash_pk %', person_crash_pk;
+
+    UPDATE ems__incidents 
+    SET 
+        _match_event_name = 'handle_person_record_created_or_updated'
```

Comment:
awesome so this is where the event is queued up to be handled by `ems_update_handle_record_match_event` which has the new event handling ready to go.